core/build.gradle

```diff
@@ -66,4 +72,17 @@ jar {
 // To remove the "-all" postfix from the produced jar file.
 shadowJar {
     archiveClassifier = null
-}
\ No newline at end of file
+}
+
+
+
+// TODO: Remove below once NullAway 0.9.10 is released
```

Comment:
Are we blocked on a NullAway 0.9.10 release here? I think it might be a bad idea to disable tests from CI, rather than holding this PR until NullAway 0.9.10 is out. But if this is a blocker for you, let me know and we can speed up the upstream release!

Reply:
Yes, all tests in `core` module except `ConfigurationTest` requires `NullAway 0.9.10`. One other approach is that I can temporarily update the CI to clone and install `0.9.10-SNAPSHOT`. Please let me know if you prefer this approach. Thank you.

Reply:
Brought back all tests since `NullAway 0.9.10` is released. bc5fe129bcb9079dff981357864167969b3f1df2
This is ready for review now.